manifests/2.4.0/opensearch-2.4.0.yml

```diff
@@ -165,3 +165,18 @@ components:
       - windows
     checks:
       - gradle:properties:version
+  - name: asynchronous-search
+    repository: https://github.com/opensearch-project/asynchronous-search.git
+    ref: '2.4'
```

Comment:
Hi could you add these two platforms?
Thanks.
```
    platforms:
      - linux
      - windows
      - ```

Reply:
Added.